src/include/network/routines.rb

```diff
@@ -743,6 +745,32 @@ def SetAllLinksDown
       interfaces.all? { |i| SetLinkDown(i) }
     end
 
+    # Calls wicked ifdown if it is the currenct backend, set the link down
+    # otherwise
+    #
+    # @param [String] name of interface to put down
+    # @see SetLinkDown
+    def SetInterfaceDown(dev_name)
+      if NetworkService.is_wicked
+        NetworkService.run_wicked("ifdown", dev_name)
```

Comment:
may be you can add a friend into this module https://github.com/yast/yast-network/blob/master/src/lib/network/wicked.rb